include/fs/fs.h

```diff
@@ -341,34 +348,34 @@ off_t fs_tell(struct fs_file_t *zfp);
  *
  * Truncates the file to the new length if it is shorter than the current
  * size of the file. Expands the file if the new length is greater than the
- * current size of the file. The expanded region would be filled with zeroes.
+ * current size of the file.  The expanded region would be filled with zeroes.
```

Comment:
"Two spaces after a period" is a style preferred by people who learned to type on typewriters and stick with fixed-width fonts on computers (raises hand).  While I use it and [science supports it](https://www.theatlantic.com/science/archive/2018/05/two-spaces-after-a-period/559304/) I don't think existing text with one space should be changed unless that becomes a project-wide standard.  Or at least a standard within the file, in which case it should probably be done in a separate white-space--only commit as is preferred for styling code.

Reply:
Actually I have altered the description and undo it not enough times and this is leftover.
But I can do unification of all the spaces after period into one style, at all this PR claims to improve the documentation...
@pabigot which way are we going then? "Two spaces after period" is it?

Reply:
Yeah, but we don't have consensus on whether two or one spaces is an improvement.  My opinion: if you didn't touch an adjacent sentence for some other reason, leave the spacing alone.  If you did, use the spacing that's used in the original version of the comment.  If it was inconsistent before, pick whichever was most often used for the content you changed, and leave the others alone (even if they remain inconsistent).

Minimal changes necessary to accomplish the task are generally preferred.

Reply:
OK